style.css

```diff
@@ -1 +1,304 @@
-/*  Add your styling here */
+@import url(https://fonts.googleapis.com/css2?family=Dancing+Script:wght@700&family=Roboto:wght@100&display=swap);
```

Comment:
Consider normalizing the styles, as you did in previous week's exercise. Default fonts in particular are not visually pleasing.